mindmeld/resource_loader.py

```diff
@@ -446,7 +446,7 @@ def _load_cached_models(self):
                 hash_val = open(file_path, "r").read()
                 classifier_file_path = file_path.split(".hash")[0]
                 if not os.path.exists(classifier_file_path):
-                    logger.warning("Could not find the serialized model")
+                    logger.info("Could not find the serialized model")
```

Comment:
I think warning is better here.

Reply:
@vijay120 So the warning is leading to failure of a test case. And also, for RoleClassifiers, when no model is required, we do not save any `entity-role.pkl` file which is causing this error.